cobra/test/test_flux_analysis.py

```diff
@@ -78,15 +68,14 @@ def do_pfba(solver):
     def test_pfba(self, model, solver):
         expression = model.objective.expression
         n_constraints = len(model.solver.constraints)
-        optimize_minimal_flux(model, solver=solver)
-        abs_x = [abs(i) for i in model.solution.x]
+        df = optimize_minimal_flux(model, solver=solver)
+        assert numpy.all([df.columns.values == ['flux', 'objective_value']])
```

Comment:
Checking the column names here, so should be identical but perhaps there is better way to do it..